ra/ra.go

```diff
@@ -1553,10 +1562,53 @@ func revokeEvent(state, serial, cn string, names []string, revocationCode revoca
 	)
 }
 
+// revokeCertificate generates a revoked OCSP response for the given certificate, stores
+// the revocation information, and purges OCSP request URLs from Akamai.
+func (ra *RegistrationAuthorityImpl) revokeCertificate(ctx context.Context, cert x509.Certificate, code revocation.Reason) error {
```

Comment:
Do you think there's any value in a unit test for this function? I don't think the existing tests have coverage for the new code.

Reply:
I was somewhat on the fence about this. There isn't really any logic here, just a bunch of calls to other services, so the test would just end up being a bunch of mocks testing if the err catches work, which feels a bit unnecessary. But I could probably be convinced otherwise, what do you think?

Reply:
I think you're right about the value of testing it at the unit level. I left this comment before seeing the integration test component. I think with that I'm less convinced its worth the trouble of mocking the error handling. I'm +1 to leaving it as-is.